src/test/scala/com.snowplowanalytics.dataflow/BucketSpec.scala

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+
+//Scala
+import org.specs2._
+import scala.collection.JavaConverters._
+
+//Dataflow testing
+import com.google.cloud.dataflow.sdk.transforms.DoFnTester
+
+//This project
+import com.snowplowanalytics.dataflow.streaming.SimpleEvent
```

Comment:
This should be `package com.snowplowanalytics.dataflow.streaming`, and no imports are necessary then.